notebooks/examples/CEMS.ipynb

```diff
@@ -0,0 +1,1130 @@
+{
```

Comment:
<p>This is a little bit dangerous -- it'll trigger the full computation of any tasks that exist in the dataframe. In this case it works okay because you've only just read it in, and the number of records is stored in the parquet file metadata, but in general this could be very computationally intensive.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>hmmm good to know.</p>